experiment/simulation/css/main.css

```diff
@@ -1 +1,228 @@
-/* You CSS goes in here */
\ No newline at end of file
+.main{
+    display: flex; 
+    justify-content: center; 
+    align-items: center;
+}
+
+.title {
+    text-align: center;
+    margin-top: 5px;
+    font-size: 2.5vw;
+    z-index: -1;
+}
+
+.nav-link ,.navbar-nav,.navbar ,.navbar-custom{
+    padding-left: 15px;
+    padding-right: 15px;
+    background-color: rgb(21,100,156) ;
+}
+.nav-link{
+    color:rgb(169,169,169);
+}
+
+.activeLink{
+    color: rgb(151, 220, 61);
+}
+.navbar-nav>li>a:focus,.navbar-nav>li>a:hover{
+    color: rgb(151, 210, 61);
+    background-color: rgb(21,100,156) ;
+}
+
+.procedureMessage {
```

Comment:
use the hyphenated naming convention for css consistently. CamelCase should be used for js. So this class will be named as procedure-message